apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1391,15 +1391,11 @@
     }
 
     // Sort according to timestamp
-    if (!reverse) {
-      msgs.sort(function(a, b) {
-        return b.timestamp - a.timestamp;
-      });
-    } else {
-      msgs.sort(function(a, b) {
-        return a.timestamp - b.timestamp;
-      });
-    }
+    var sortFunc = reverse ?
+      (a, b) => b.timestamp - a.timestamp :
+      (a, b) => a.timestamp - b.timestamp;
```

Comment:
it was done the wrong way... I took the opportunity to rewrite this part as well.